develop.sh

```diff
@@ -5,6 +5,12 @@ if [[ ! -d "docker" ]]; then
     exit -1
 fi
 
+function invoke_docker_compose_spark {
+	docker-compose -f docker/docker-compose.dev.yml \
```

Comment:
Can we rename this docker-compose file to docker-compose.spark.yml

Reply:
Yes, we can.